packages/apps/intervention/src/i18n/en.json

```diff
@@ -9,5 +9,14 @@
   "Intervention_Status_Planned": "Planned",
   "Intervention_Status_Started": "Started",
   "Intervention_Status_Suspended": "Suspended",
-  "Intervention_SliceAction_FetchIntervention": "fetch intervention"
+  "Intervention_CustomerPark": "Customer Park",
+  "Intervention_Equipments": "Equiments",
+  "Intervention_Status_InService": "In Service",
+  "Intervention_Customer": "Customer",
```

Comment:
Move Customer above InService as it was done in fr translations. When keys are linked, it is better to leave them together in the translations files

Also, please remove uppercase on service in the translation value